_posts/2017-06-27-CivilAssetForfeitureReform.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: "Illinois Passes Civil Asset Forfeiture Reform"
```

Comment:
This makes it sound like HB 303 is now law - my understanding is that this was not the case since it hasn't be signed by the governor. Instead, this should say "Illinois Legislature Passes Civil Asset Forfeiture Reform", no? Correct me if I'm wrong here @freddymartinez9